3. Installation/3. Docker Containers/systemd/README.md

```diff
@@ -4,20 +4,101 @@ Deploy [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) to Linux that ru
 
 ## Prerequisite
 
-You need to have [docker](https://docs.docker.com/linux/started/) and [docker-compose](http://docs.docker.com/compose/) installed.
+You need to have [docker](https://docs.docker.com/linux/started/) installed.
 
 ## How to run Rocket.Chat on systemd.
 
+Create three systemd files. One for the database, one for the database replica and one for rocketchat.
+
+mongo.service:
+```
+[Unit]
+Description=mongo
+Requires=docker.service
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=dockeruser
+Restart=always
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill mongo
+ExecStartPre=-/usr/bin/docker rm mongo
+ExecStartPre=-/usr/bin/docker pull mongo:3.2
+
+
+ExecStart=/usr/bin/docker run \
+    --name #name for dockercontainer# \
+    -v .../path/to/data/db:/data/db \
+    -v .../path/to/data/dump:/data/dump \ <--optional
+    --net=rocketchat_default \
+    mongo:3.2 \
+    mongod --smallfiles --oplogSize 128 --replSet rs0
+
+ExecStop=-/usr/bin/docker kill mongo
+ExecStop=-/usr/bin/docker rm mongo
+```
+
+mongo-init-replica.service:
 ```
-git clone https://github.com/RocketChat/DockerFiles.git
-cd systemd
-cp ./*@* /etc/systemd/system && systemctl daemon-reload
-mkdir -p /data/domains/
-cd ..
-cp GenericLinux /data/domains/example.org
-vi /data/domains/example.org/docker-compose.yml # modify `ROOT_URL` and `MAIL_URL`
-systemctl enable universal@example.org
-systemctl start universal@example.org
+[Unit]
+Description=mongo-init-replica
```

Comment:
You do not need to run it everytime. Thats right. Perhaps it is better to make this a one time command in the beginning.